model-optimizer/unit_tests/mo/moc_frontend/moc_mean_scale_test_actual.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+from argparse import Namespace
+
+from mo.utils.error import Error
+
+import numpy as np
+
+
+try:
+    # pylint: disable=no-name-in-module,import-error
+    import ngraph as ng
+    from ngraph import Function, PartialShape
+    from ngraph import function_to_cnn
+    from mo.moc_frontend.mean_scale import process_mean_scale
+
+except Exception:
+    print("No ngraph API available,"
+          "ensure to set correct PYTHONPATH when running these tests")
+    raise
+
+
+def create_function2(shape1=[2, 2], shape2=[2, 2]):
+    input1 = ng.parameter(shape1, dtype=np.float32, name="input1")
+    relu1 = ng.relu(input1)
+    res1 = ng.result(relu1, "res1")
+    input2 = ng.parameter(shape2, dtype=np.float32, name="input2")
+    relu2 = ng.relu(input2)
+    res2 = ng.result(relu2, "res2")
+    function = Function(results=[res1, res2], parameters=[input1, input2], name="TestFunction")
+    return function
+
+
+def check_scale_constant(node, expected, shape=None):
+    const_node = node.input(1).get_source_output().get_node()
+    assert const_node.get_type_name() == 'Constant'
+    if node.get_type_name() == 'Divide':
+        assert (const_node.get_vector() == expected).all()
+    else:
+        assert (const_node.get_vector() == 1. / expected).all()
+    if shape:
+        assert const_node.shape == PartialShape(shape)
+
+
+def check_mean_constant(node, expected, shape=None):
+    const_node = node.input(1).get_source_output().get_node()
+    assert const_node.get_type_name() == 'Constant'
+    if node.get_type_name() == 'Subtract':
+        assert (const_node.get_vector() == expected).all()
+    else:
+        assert (const_node.get_vector() == -expected).all()
+    if shape:
+        assert const_node.shape == PartialShape(shape)
+
+
+class TestMeanScaleMOC(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_scale_single_value(self):
+        argv = Namespace(mean_scale_values=None, scale=2.0)
+        function = create_function2()
+        ie_network = function_to_cnn(function)
+        process_mean_scale(network=ie_network,
+                           ngraph_function=function,
+                           argv=argv)
+        for param in function.get_parameters():
+            op_node = list(param.output(0).get_target_inputs())[0].get_node()
+            assert op_node.get_type_name() == 'Divide' or op_node.get_type_name() == 'Multiply'
```

Comment:
@GlebKazantaev I see that after PR #6871 in tests it would probably be better to `ApplyMOCTransformations` and verify that resulting graph has 'Multiply/Add' pair always. What do you think?